cbr-service-profile/README.md

```diff
@@ -1,3 +1,74 @@
-# CBR Rule Profile
+# CBR Service Profile
 
-Accepts a list of VPC crns / service references to create CBR zones and a list of target services, to create the rule matching these profiles.  It supports to target the service using name, account id, tags, resource group.
+This profile creates a list of CBR zones and rules. Accepts a list of VPC CRNs / service references to create CBR zones and a list of target services to create the rule matching these profiles. It supports to target the service using name, account id, tags, resource group.
```

Comment:
```suggestion
The CBR service profile creates a list of zones and rules for context-based restrictions. The example accepts a list of VPC CRNs or service references to create CBR zones, and also accepts a list of target services to create the rules that match these profiles. The example supports identifying the service by using name, account ID, tags, or resource groups.
```

Some questions that aren't clear to me:
- Does the list of VPC CRNs actually "create" the zones, or is there another verb that identifies what that list is for?
- Similar question about the list of services: do it "create" the list or something else?
- Did I get the last sentence right? That you can "identify" the service in various ways rather than target? If not, what does "target" mean?
- Which "service" is targeted/identified in the last sentence? Other mentions of services are plural.

Reply:
Hi @SirSpidey, I have tried answering the below questions, @vburckhardt please verify if I am missing something.
- Does the list of VPC CRNs actually "create" the zones, or is there another verb that identifies what that list is for?
  - List of VPC CRNs is passed to the zone module to create the zone containing VPCs.
  
- Similar question about the list of services: do it "create" the list or something else?
  - List of service references is passed as a list to the zone module to create the zone for each service reference.
 
- Did I get the last sentence right? That you can "identify" the service in various ways rather than target? If not, what does "target" mean?
  - Yes, this is correct, basically we want to convey that we can scope the restrictions by using name, account ID, tags, or resource groups for the target service.

- Which "service" is targeted/identified in the last sentence? Other mentions of services are plural.
  - Here we are referring "service" as target services for which the rules are being created.

Reply:
```suggestion
The CBR service profile creates a list of zones and rules for context-based restrictions. You pass a list of VPC CRNs or service references to create the CBR zones. You also pass a list of target services to create the rules that match these profiles. You can identify the services by name, account ID, tags, or resource groups.
```

Questions are answered:
- [x] _(OK, leave as create)_ Does the list of VPC CRNs actually "create" the zones, or is there another verb that identifies what that list is for?
- [x] _(OK, leave as create)_ Similar question about the list of services: does it "create" the list or something else?
- [x] Did I get the last sentence right? That you can "identify" the service in various ways rather than target? If not, what does "target" mean?
- [x] _make services plural_ Which "service" is targeted/identified in the last sentence? Other mentions of services are plural.